target/linux/ath79/image/mikrotik.mk

```diff
@@ -35,3 +35,18 @@ define Device/mikrotik_routerboard-wap-g-5hact2hnd
   SUPPORTED_DEVICES += rb-wapg-5hact2hnd
 endef
 TARGET_DEVICES += mikrotik_routerboard-wap-g-5hact2hnd
+
+define Device/mikrotik_sxt5n
```

Comment:
I will introduce mikrotik_nand recipe soon (or maybe already have when you read this). I don't think a sxt5n is still needed then, and would move the remaining parameters to the device node.